src/services/web-monitoring-db.ts

```diff
@@ -72,6 +72,18 @@ export default class WebMonitoringDb {
             .then(data => parsePage(data.data));
     }
 
+    getPagesByDomains (domains: string[]): Promise<Page[]> {
+        const daysAgo = 3;
+        const dateEarlier = new Date(new Date().setDate(new Date().getDate() - daysAgo)).toISOString();
+        const fetches = domains.map(domain => {
+            return fetch(this.createUrl(`pages?site=${encodeURI(domain)}&capture_time=${dateEarlier}..`))
```

Comment:
You should use `encodeURIComponent` here—it’s *slightly* different from `enocdeURI` and is more correct in this situation.